packages/tools/piw-utils-internal/src/constants/Colors.ts

```diff
@@ -79,15 +85,17 @@ export const ColorsLight: ColorsType = {
 export const ColorsDark: ColorsType = {
     background: {
         topBar: {
-            data: "#C2E2FF",
-            standard: "#323232"
+            data: colorWithOpacity("#3A65E5", 20),
+            standard: colorWithOpacity("#646464", 20)
         },
```

Comment:
It won't work if you use colors that have opacity.

Reply:
Reverted back to only color.